web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/GetCertificatesAvailableFunctionsImpl.java

```diff
@@ -30,8 +30,6 @@
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
All the tests look good. But you could add tests in the old implementation GetCertificatesAvailableFunction to verify that if your service returns a resource link it's added to the result. If you add this test you would be testing lines 250 and 251. You can look at the tests: shallIncludeSRSMinimizedView and shallExcludeSRSMinimizedView for inspiration.